app/core/utils.js

```diff
@@ -209,10 +209,9 @@ utils.processPatientData = (data, queryParams, timePrefs, bgPrefs) => {
   }
   // but otherwise we use the timezone from the most recent upload metadata obj
   else {
-    // @todo: need to figure out a way to refactor this bit of shenanigans
-    // comp.setState({
-    //   timePrefs: timePrefsForTideline
-    // });
```

Comment:
Yea I def want to refactor this. Could tackle shortly. Do not think it is too difficult to tidy up.